doc/asciidoc/algorithms/alpha/alpha-particle-filtering.adoc

```diff
@@ -0,0 +1,99 @@
+[[algorithms-particle-filtering-centrality]]
+[.alpha]
+= Particle Filtering
+
+[abstract]
+--
+This section describes the Particle Filtering algorithm in the Neo4j Graph Data Science library.
+--
+
+Particle filtering measures the proximity between source nodes and other nodes in the graph.
```

Comment:
Particle filtering methods produce an approximation of the Personalized Page Rank (PPR) score with sufficient precision while exploiting what graph databases architectures are already optimized for: navigating local connections.

Personalized Page Rank (PPR) is a particularly important task to support search and exploration within graphs. 
PPR computes the proximity between query nodes and other *local* nodes in the graph.
In practice, given a small set of query entities, PPR returns a ranked list of other relevant entities based on a computed random-walk proximity to the query nodes.
This is used, among others, for entity exploration, query expansion, and product recommendation.